train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -43,7 +58,8 @@ private void enforceSpeedLimit() {
 
 	@Override
 	public void setJoystickPosition(int joystickPosition) {
-		this.step = joystickPosition;		
+		this.step = joystickPosition;
+		followSpeed();
```

Comment:
Ezzel neked helyesen futott le a teszt?

Reply:
Marmint így, hogy a setJoystickPosban is van egy followSpeed() hivas

Reply:
Fixed.